docs/plugins/analysis-kuromoji.asciidoc

```diff
@@ -172,6 +172,8 @@ PUT kuromoji_sample
   }
 }
 
+GET _cluster/health?wait_for_status=yellow
```

Comment:
It is being discussed in a more general way:
https://github.com/elastic/elasticsearch/issues/9126

I'm not sure I *want* to do it with some magic in the testing framework because it makes the docs not copy-and-paste-able.